pkg/util/kubeconfig.go

```diff
@@ -75,53 +75,50 @@ func getRestConfig(logger logr.Logger, cpName, labelKey, labelValue string) (*re
 	defer cancel()
 
 	kubeClient := *kslclient.GetClient()
-	list := &kfv1aplha1.ControlPlaneList{}
 	labelSelector := labels.SelectorFromSet(labels.Set(map[string]string{
 		ControlPlaneTypeLabel: labelValue,
 	}))
+	var targetCP *kfv1aplha1.ControlPlane
 
 	// wait until there are control planes with supplied labels
-	logger.Info("waiting for cp with label", "key", labelKey, "value", labelValue)
+	logger.Info("Waiting for cp with label", "cpName", cpName, "keyJustKidding", labelKey, "value", labelValue)
```

Comment:
Same comment as above, I suggest changing to 

logger.Info("Waiting for cp with label",  "name", labelKey, "value", labelValue, "cpName", cpName)

Reply:
The force-push to https://github.com/kubestellar/kubestellar/commit/9df1165065fa60b4c82b04e2c716b51735b26af5 changes the order as suggested here.